pyproject.toml

```diff
@@ -45,20 +45,20 @@ djangocms-admin-style = "2.0.0"
 djangocms-apphook-setup = "0.4.1"
 djangocms-attributes-field = "2.1.0"
 djangocms-blog = "^1.2"
-djangocms-bootstrap4 = "3.0.0"
+djangocms-bootstrap4 = { git = "https://github.com/django-cms/djangocms-bootstrap4", rev = "2fc7f14c2b7b53882faf6830289b5052f775965f" }
```

Comment:
1. Where does [that commit](https://github.com/django-cms/djangocms-bootstrap4/commit/2fc7f14c) come from?
2. Is the goal to remove `mptt`?

<details><summary>Concerns</summary>

It has no clear paper trail, thus future attempts to update could be difficult.

</details>

<details><summary>My Failed Attempt to Find the Answers</summary>

I **did** find:
1. three parent commits leads to the merge commit of [django-cms/djangocms-bootstrap4#160](https://github.com/django-cms/djangocms-bootstrap4/pull/160) (so I know `2fc7f14c` is is just a little ahead of `master` from 2022-12-26)
3. [the diff](https://github.com/django-cms/djangocms-bootstrap4/compare/master...2fc7f14) (at the very bottom) shows removal of `mptt`
4. the commit in a manual search for commits in [branches](https://github.com/django-cms/djangocms-bootstrap4/branches) that are ahead of `master`

I did **not** find a more useful identifier:

```sh
➜  djangocms-bootstrap4 git:(master) git branch -a --contains 2fc7f14c2b7b53882faf6830289b5052f775965f
error: no such commit 2fc7f14c2b7b53882faf6830289b5052f775965f
➜  djangocms-bootstrap4 git:(master) git name-rev 2fc7f14c2b7b53882faf6830289b5052f775965f
Could not get object for 2fc7f14c2b7b53882faf6830289b5052f775965f. Skipping.
```

</details>

Reply:
Also, Core might need:
- https://github.com/django-cms/djangocms-bootstrap4/pull/126
- **or** https://github.com/django-cms/djangocms-frontend (the future of `django-cms/djangocms-bootstrap…`)
	<sup>supposedly backwards-compatible, but maybe not given [my overwrite of `bootstrap4_djangocms_link/cms_plugins.py`](https://github.com/TACC/Core-CMS/blob/v3.12.0-beta.2/taccsite_cms/contrib/bootstrap4_djangocms_link/cms_plugins.py)</sup>

Reply:
**Update**: The https://github.com/django-cms/djangocms-frontend has a "management command to migrate from djangocms-bootstrap4". Good news, but using it seems like a second major change that could be done after this. I am more willing to use a vague revision, knowing that https://github.com/django-cms/djangocms-frontend is a foreseeable future.